java/arcs/core/data/expression/Expression.kt

```diff
@@ -360,11 +360,11 @@ sealed class Expression<T> {
      * @param E the type of the [qualifier] expression if any
```

Comment:
Don't forget to remove @param from all of the places you removed the params.

Reply:
Thanks! Done!